.github/workflows/docker-release.yaml

```diff
@@ -464,10 +464,10 @@ jobs:
       fail-fast: false
       max-parallel: 4
       matrix:
-        dse-version: [6.9.0, 6.9.1, 6.9.2, 6.9.3]
+        dse-version: [6.9.0, 6.9.1, 6.9.2, 6.9.3, 6.9.4]
         image-base: [jdk11]
         include:
-          - dse-version: 6.9.3
+          - dse-version: 6.9.4
```

Comment:
**comment:**
As of DSE 6.9.4, the k8ssandra management API is bundled in and the `dse_server` images produced should work with the K8ssandra stack. Going forward, we shouldn't have to update the DSE 6.9 images any more in this project. However, I still need to figure out a testing strategy as we'll now have a small chicken-and-egg problem.

Reply:
Are those dse-server updates pushed to datastax/dse-mgmtapi-6_8 also?

Reply:
If you mean DSE 6.8 server images, the answer is "no". Only DSE 6.9 images have the k8ssandra Management API, and only since 6.9.3. However, the 6.9.3 image that was built did not have the changes necessary for it to load the Management API agent, so it's effectively broken in that image. That was fixed in 6.9.4.

Reply:
No, I mean 6.9 also. We load 6.9 images from ``datastax/dse-mgmtapi-6_8`` repository. So we need 6.8 and 6.9 images in that repository to use them, the ``dse-server`` repository is not used by us.